api/src/projects/ProjectService.ts

```diff
@@ -9,6 +11,8 @@ export interface Project {
   description_link: string;
   org_url: string;
   logo_link: string;
+  mvp_link: string;
```

Comment:
This interface could use some comments. Not clear to me what `po_link` is for example but probably just good practice to comment on every field here.